en/simulation/hitl.md

```diff
@@ -113,26 +113,24 @@ Follow the appropriate setup steps for your simulator in the following sections.
 
 > **Note** Make sure *QGroundControl* is not running!
 
-1. Update the environment variables: 
+1. Build PX4 with the Gazebo plugins although we only need the Gazebo part:
    ```sh
    cd <Firmware_clone>
-   make px4_sitl_default gazebo
+   DONT_RUN=1 make px4_sitl_default gazebo
    ```
-   In a new terminal, run:
-   ```sh
-   source Tools/setup_gazebo.bash $(pwd) $(pwd)/build/px4_sitl_default
-   ```
-  
 1. Open the vehicle model's sdf file (e.g. **Tools/sitl_gazebo/models/iris/iris.sdf**).
-1. Under the `mavlink_interface plugin` section, change the `serialEnabled` and `hil_mode` parameters to `true`. 
+1. Under the `mavlink_interface plugin` section, change the `serialEnabled` and `hil_mode` parameters to `true`.
```

Comment:
From your comment above https://github.com/PX4/Devguide/pull/778/files#r280022112 is it worth noting that the sdf file is autogenerated and that it needs to be re-edit or cached. 

I was thinking under the image below you might have
```
 ![HIL Parameters](../../assets/simulation/gazebo_sdf_model_hil_params.png)
> **Note** SDF files, including this one, may be autogenerated from XXXXX. If this is the case, then you will need to either keep a copy of your changed file or re-edit it for every build. 
```

Reply:
Yes, I'll add that, thanks.